docs/java-api/client.asciidoc

```diff
@@ -59,18 +59,18 @@ Client client = TransportClient.builder().settings(settings).build();
 
 The Transport client comes with a cluster sniffing feature which
 allows it to dynamically add new hosts and remove old ones.
-When sniffing is enabled the transport client will connect to the nodes in its
-internal node list, which is built via calls to addTransportAddress.
+When sniffing is enabled, the transport client will connect to the nodes in its
+internal node list, which is built via calls to `addTransportAddress`.
 After this, the client will call the internal cluster state API on those nodes
 to discover available data nodes. The internal node list of the client will
 be replaced with those data nodes only. This list is refreshed every five seconds by default.
-Note that the IP addresses the sniffer connects to are the ones declared as the 'publish'
+Note that the IP addresses, the sniffer connects to, are the ones declared as the 'publish'
```

Comment:
these commas are incorrect.  better to say "Note that the IP addresses that the sniffer connects to are"